eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/eh-asa-perfmon.csproj

```diff
@@ -4,11 +4,13 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <RootNamespace>StreamingAtScale</RootNamespace>
+    <LangVersion>preview</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.1" />
     <PackageReference Include="CommandLineParser" Version="2.5.0" />
-    <PackageReference Include="Microsoft.Azure.EventHubs" Version="3.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Shouldn't this be System.Text.Json?

Reply:
The origin sample uses Newtonsoft.Json and I didn't change it. 
There occurred such changes that Newtonsoft.Json dependency was removed when I update EventHub SDK to Track2. So I have to re-install Newsoft.Json manually.

Reply:
Does it need Newtonsoft or can we move that to System.Text.Json?

Reply:
Yes, there is an API called "JsonConvert", so we need Newtonsoft.Json. I tried to move it to System.Text.Json, but because of the huge difference, it took a lot of time. I suggest we can change it later.

Reply:
Is it good to merge?

Reply:
I've asked Jesse to review.

Reply:
Any update on this?